### PR TITLE
Add feature suggestion issue template

### PR DESCRIPTION
**Pull request by jonaro00**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/FEATURE-SUGGESTION.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+0name: Feature suggestion
```
Comment (oddgrd) [inline .github/ISSUE_TEMPLATE/FEATURE-SUGGESTION.yml:1]: "0name" is not a valid key. I'm also wondering if it should be called "Feature Request", but that's a bit of a nit and either is fine.
```suggestion
name: Feature suggestion
```

File: .github/ISSUE_TEMPLATE/FEATURE-SUGGESTION.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    attributes:
+      label: Suggestion or Example of how the feature would be used
+      description: What should the feature look like?
+  - type: checkboxes
+    id: duplicate
+    attributes:
+      label: Duplicate declaration
+      description: Please confirm that you are not creating a duplicate issue.
+      options:
+        - label: I have searched the issues and there are none like this.
```
Comment (oddgrd) [inline .github/ISSUE_TEMPLATE/FEATURE-SUGGESTION.yml:29]: Should this be something like "... and this feature has not been requested before."?

**Review by oddgrd — APPROVED** [review]
Thanks so much @jonaro00, this is great. Just two very small comments and this is good to merge :partying_face:

File: .github/ISSUE_TEMPLATE/FEATURE-SUGGESTION.yml (hunk 1 above)
Comment (jonaro00) [reply .github/ISSUE_TEMPLATE/FEATURE-SUGGESTION.yml]: Not sure how that got there 😄

**Comment by ivancernja** [comment]
lgtm!

**Pull request merged**